src/game/scenes/arena.c

```diff
@@ -576,7 +573,15 @@ void arena_har_defeat_hook(int player_id, scene *scene) {
         }
     }
     chr_score *score = game_player_get_score(game_state_get_player(gs, other_player_id));
-    object_select_sprite(local->player_rounds[other_player_id][score->rounds], 0);
+    object *round_token = game_state_find_object(gs, local->player_rounds[other_player_id][score->rounds]);
+    if (!round_token) {
+        DEBUG("could not found round token [%d][%d]", other_player_id, score->rounds);
+    } else {
+        DEBUG("found round token [%d][%d] at %d", other_player_id, score->rounds, local->player_rounds[other_player_id][score->rounds]);
+    }
+    object_set_sprite_override(round_token, 0);
```

Comment:
This line crashes on tournament fight end.
```
#0  0x00006479c5734731 in object_set_sprite_override (obj=0x0, override=0) at /home/martti/gh/openomf/src/game/protos/object.c:527
#1  0x00006479c573a987 in arena_har_defeat_hook (player_id=1, scene=0x6479c7b1a220) at /home/martti/gh/openomf/src/game/scenes/arena.c:582
#2  0x00006479c573b0ad in arena_har_hook (event=..., data=0x6479c7b1a220) at /home/martti/gh/openomf/src/game/scenes/arena.c:695
```

Reply:
Fixed